cli/CHANGELOG.md

```diff
@@ -47,6 +47,8 @@ _Released 04/17/2023_
 
 **Features:**
 
+- Cypress now allows targeted Content-Security-Policy and Content-Security-Policy-Report-Only header directive stripping from requests via the [`stripCspDirectives`](https://docs.cypress.io/guides/references/configuration#stripCspDirectives) config option. Addresses [#1030](https://github.com/cypress-io/cypress/issues/1030). Addressed in [#26483](https://github.com/cypress-io/cypress/pull/26483).
```

Comment:
Since we've had a couple releases in the interim, can you move this up under a new `12.3.0` heading, release date `05/23/2023`? I promise we'll do everything we can to get this merged in time for that release 😅